packages/embeddings/tests/Unit/Adapter/Cache/CacheEmbeddingAdapterTest.php

```
@@ -152,24 +152,24 @@ public function testEmbedWithDifferentInputsGeneratesDifferentCacheKeys(): void
         $cacheItem1 = $this->prophesize(CacheItemInterface::class);
         $cacheItem1->isHit()->willReturn(false);
-        $cacheItem1->set(Argument::any())->will(fn () => $cacheItem1->reveal());
+        $cacheItem1->set(Argument::any())->will(static fn () => $cacheItem1->reveal());
 
         $cacheItem2 = $this->prophesize(CacheItemInterface::class);
         $cacheItem2->isHit()->willReturn(false);
-        $cacheItem2->set(Argument::any())->will(fn () => $cacheItem2->reveal());
+        $cacheItem2->set(Argument::any())->will(static fn () => $cacheItem2->reveal());
 
-        $this->cacheItemPool->getItem(Argument::any())->will(function ($args) use ($cacheItem1, $cacheItem2) {
+        $this->cacheItemPool->getItem(Argument::any())->will(static function ($args) use ($cacheItem1, $cacheItem2) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$args'. (undefined)

(UnusedFormalParameter)